app.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const app = express();
-const { getCategories, getReview, patchReview, getUsers } = require("./controllers/controllers")
+const { getCategories, getReview, patchReview, getUsers, getReviews } = require("./controllers/controllers")
 
```

Comment:
You have one controllers file and one models file for all the requests. This isn't following the MVC pattern effectively and you should have a different file to handle the different types of requests. For example one each for review, comments, categories and users. 

I like to think that if some data has its own table in the database, then it should also have its own model and controller files